src/app/cic/utils.js

```diff
@@ -0,0 +1,9 @@
+function formatDate(date) {
+  const datePart = date.match(/\d+/g),
+  year = datePart[0].substring(0,4),
+  month = datePart[1], day = datePart[2];
```

Comment:
could this be split into separate lines with semi colon ending them
Also could unit test be added for it please

Reply:
This has been done, jest has been added to package.json - for the time being I changed the yarn test command from mocha to jest but if we do need mocha I can add a separate jest test command